### PR TITLE
restore proper lowercase/plural CRD resource

### PR DESCRIPTION
**Pull request by sseago**
Fixes https://github.com/vmware-tanzu/velero/issues/2948

This commit restores the proper resource string
"customresourcedefinitions" for CRD. The prior change to
"CustomResourceDefinition" was made because this was being used
in another place to populate the CRD "Kind" field in
remap_crd_version_action.go -- there, just use the correct Kind
string instead of pulling from Resource. Using the Kind string in place of Resource was breaking the "wait for CRDs to be ready" functionality, as well as the "refresh discovery helper after loading CRDs" bit.

### CODE REVIEW

File: changelogs/unreleased/2949-sseago
Comment (nrb) [inline changelogs/unreleased/2949-sseago:1]: I might rephrase this to talk about the wait behavior; from a user standpoint the lowercase/plural doesn't mean much.

**Review by ashish-amarnath — COMMENTED** [review]
Thanks for the bug report and the fix!
This LGTM. Waiting for the changelog update.

**Comment by nrb** [comment]
> This LGTM. Waiting for the changelog update.

Looks like the changelog update has been added, I just didn't resolve the conversation.

File: changelogs/unreleased/2949-sseago
Comment (sseago) [inline changelogs/unreleased/2949-sseago:1]: @ashish-amarnath I believe this update is in line with what @nrb had suggested. If I misunderstood, or if more than a one-line summary is better, I can change it again.

**Review by nrb — APPROVED** [review]

**Review by zubron — APPROVED** [review]
Thanks, @sseago! These changes look good to me. I'm going to hold off on merging though and let @ashish-amarnath have the final approval.

**Comment by zubron** [comment]
Merging as Ashish is out of office.

**Comment by rahul-mereddy** [comment]
should I need to include flag --include-cluster-resources during the backup and restore in velero version 1.8.0?

During the recent velero backup and restore on GKE cluster we are seeing CRD's are missing during the restore.

how do I know if the velero backup or restore did the crd's restored?

**Comment by sseago** [comment]
@rahul-mereddy if include-cluster-resources is `true`, then all cluster-scoped resources will be backed up, regardless of whether they're relevant to the namespaces being backed up, which means all CRDs should be included. If it's set to `false` then no cluster-scoped resources will be included, including CRDs. If it's `nil` (the default value), then only relevant cluster-scoped resources will be included. For CRDs, this means that if a CR for `mytype.myorg.org` is backed up, then the `mytype.myorg.org` CRD will be backed up as well, but `irrelevanttype.myorg.org` will not be backed up.

**Pull request merged**
